<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>queryStu</class>
 <widget class="QDialog" name="queryStu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查询学生</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,8,1">
     <item>
      <widget class="QComboBox" name="cbb_method">
       <item>
        <property name="text">
         <string>请选择查询方式</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按姓名</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按学号</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>按院系</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_cnt"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_search">
       <property name="text">
        <string>开始查询</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="textElideMode">
      <enum>Qt::ElideRight</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
